TestsFoundation/TestsFoundation/DataSeeding/SeedClient.swift

```diff
@@ -256,6 +256,19 @@ public struct SeedClient {
         return makeRequest(PostFileUploadRequest(fileURL: url, target: target), with: user.token)
     }
 
+    @discardableResult
+    public func uploadFile(
+        url: URL,
+        named name: String? = nil,
+        as user: AuthUser
+    ) -> APIFile {
+        let target = makeRequest(PostFileUploadTargetRequest(
+            target: .myFiles,
+            body: .init(name: name ?? url.lastPathComponent, on_duplicate: .rename, parent_folder_id: nil)
+        ), with: user.token)
+        return makeRequest(PostFileUploadRequest(fileURL: url, target: target), with: user.token)
+    }
```

Comment:
Does the above `uploadFile` not work for AV? I would assume we want user's assignment submissions to not require using their personal storage quota.